### PR TITLE
Move to 50ns L1 menu

### PR DESCRIPTION
**Pull request by perrotta**
Put 50ns specific L1 seeds (and dedicated customization) into the 50ns HLT menu for Run2

### CODE REVIEW

**Comment by perrotta** [comment]
please test

**Comment by fwyzard** [comment]
rebased on top of CMSSW_7_4_X, and history cleanup

**Comment by fwyzard** [comment]
@cmsbuild please test

**Comment by fwyzard** [comment]
(even though there are no significant differences between Andrea's original commits and the latest ones)

**Comment by Martin-Grunewald** [comment]
Oh man

**Comment by fwyzard** [comment]
@Martin-Grunewald ?
I needed to clean up the history, otherwise rebasing #8411 on top of this would have been too complicated.

**Comment by Martin-Grunewald** [comment]
And you broke my area now I need to start again to put the required fixes!

**Comment by fwyzard** [comment]
Sorry :-(

**Comment by fwyzard** [comment]
Feel free to force-update this branch frum your current area, I will take care of propagating the changes to #8445

**Comment by fwyzard** [comment]
Or, you could make the fixes directly on the #8445 branch ?

**Comment by Martin-Grunewald** [comment]
I get too many merge conflicts with the 8445 branch

**Comment by fwyzard** [comment]
Apologies again :-(
Would it help if I revert this branch to Andrea's version ?

**Comment by Martin-Grunewald** [comment]
I have made a new area with cms-tsg-storm:move_to_50ns_L1_menu and will update
here . Merging in your _v2 branch gives me a lot of merge conflicts, so I'll use and update
the current one.

**Comment by fwyzard** [comment]
OK, then I won't touch anything for the moment, and will update the v2 branch from this one once you are done.

If for any reason you need to recreate cms-tsg-storm:move_to_50ns_L1_menu as was first requested by Andrea, this should work:
```
git checkout move_to_50ns_L1_menu
git reset --hard 90812d9
```

.A

**Comment by perrotta** [comment]
There are broken seeds to be fixed.

But I am completely lost now with all what you did with this PR: I don't even see my commits any more...
Ok, since you are apparently working on it, I let you take over and finish with it,

**Comment by Martin-Grunewald** [comment]
/dev/CMSSW_7_4_0/50nsHLT/V14 should fix that... testing

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
@fwyzard @perrotta 
should be fine now - fingers crossed

**Comment by perrotta** [comment]
@Martin-Grunewald : not finished yet.
InputTag must be fixed for 
HLT_Ele27_eta2p1_WP75_Gsf_CentralPFJet30_BTagCSV07_v
[HLT_Ele27_eta2p1_WP75_Gsf_TriCentralPFJet30_v
[HLT_Ele27_eta2p1_WP75_Gsf_TriCentralPFJet50_40_30_v
HLT_Ele32_eta2p1_WP75_Gsf_CentralPFJet30_BTagCSV07_v
[HLT_Ele32_eta2p1_WP75_Gsf_TriCentralPFJet30_v
HLT_Ele32_eta2p1_WP75_Gsf_TriCentralPFJet50_40_30_v

I can fix it, but I see that ConfDB is locked, and I don't know whether you already ended your turn

**Comment by Martin-Grunewald** [comment]
Hmm, there are gazillions on unresolved input tags (which are unused),
what specifically do you mean?

**Comment by perrotta** [comment]
L1filters, which have in inputs wrong L1s modules

**Comment by Martin-Grunewald** [comment]
OK, go ahead

**Comment by Martin-Grunewald** [comment]
(I really wonder what the proponents actually test when they submit)

File: Configuration/HLT/python/autoHLT.py [hunk 1]
```diff
@@ -2,9 +2,10 @@
 #   cmsDiver.py hlt -s HLT:@relval
 
 autoHLT = {
-  'frozen50ns' : 'GRun',
+  'frozen50ns' : 'GRun50ns',
```
Comment (Martin-Grunewald) [inline Configuration/HLT/python/autoHLT.py]: Change GRun50ns to 50nsGRun

File: Configuration/HLT/python/autoHLT.py [hunk 2]
```diff
@@ -2,9 +2,10 @@
...
   'frozen25ns' : 'GRun',
+  'relval50ns' : 'GRun50ns',
```
Comment (Martin-Grunewald) [inline Configuration/HLT/python/autoHLT.py]: Change GRun50ns to 50nsGRun

**Comment by perrotta** [comment]
Fixes in /dev/CMSSW_7_4_0/50nsHLT/V16

Martin, if you are still around can you fix the PR yourself? I am at home and with my connection it was already a pain working with the ConfDB GUI...

**Comment by Martin-Grunewald** [comment]
OK, will do!

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
BTW, this PR also contains the -6% adjustment in the SiStrip ClusterChargeCuts (#8397) for HLT.

**Comment by Martin-Grunewald** [comment]
+1

**Comment by fwyzard** [comment]
I've updated #8445 to match #8439 as
of c014d7d3c7049a89abaeeef701a2af7d6842cebf .

Assuming Jenkis tests pass for both, if you agree to merge #8445 then DO
NOT merge #8439 before, or it will result in merge conflicts.

.A

On 21 March 2015 at 00:54, Martin Grunewald <notifications@github.com>
wrote:

> +1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8439#issuecomment-84197127>.
>



-- 
http://youtu.be/oshus9DxEG8

**Comment by perrotta** [comment]
Since #8445 had been merged already, this PR is already fully contained and we can close it.

**Pull request closed without merging**